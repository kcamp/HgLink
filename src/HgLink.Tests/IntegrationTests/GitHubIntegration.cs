// --------------------------------------------------------------------------------------------------------------------
// <copyright file="GitHubIntegration.cs" company="CatenaLogic">
//   Copyright (c) 2014 - 2014 CatenaLogic. All rights reserved.
// </copyright>
// --------------------------------------------------------------------------------------------------------------------


namespace HgLink.Tests.IntegrationTests
{
    using NUnit.Framework;

    [TestFixture, Explicit]
    public class GitHubIntegration : IntegrationTestBase
    {
        public const string Url = "https://github.com/CatenaLogic/GitLinkTestRepro";
        public const string Directory = @"C:\Source\GitLinkTestRepro_GitHub";

        [Test]
        public void CorrectlyUpdatesPdbFiles()
        {
            const string directory = Directory;
            const string configurationName = "Release";

            var result = RunGitLink(directory, Url, "master", configurationName);

            Assert.AreEqual(0, result);

            VerifyUpdatedPdbs(directory, configurationName);
        }
    }
}