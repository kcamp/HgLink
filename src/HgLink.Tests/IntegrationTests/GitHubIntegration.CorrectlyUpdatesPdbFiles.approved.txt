SRCSRV: ini ------------------------------------------------
VERSION=2
SRCSRV: variables ------------------------------------------
SRCSRVVERCTRL=https
SRCSRVTRG=https://raw.github.com/CatenaLogic/GitLinkTestRepro/8f1bebc3b06afc67e06d9c482f5871a3cbc1f001/%var2%
SRCSRV: source files ---------------------------------------
C:\Source\GitLinkTestRepro_GitHub\src\CSharpTestLibrary\Class1.cs*src/CSharpTestLibrary/Class1.cs
C:\Source\GitLinkTestRepro_GitHub\src\CSharpTestLibrary\Properties\AssemblyInfo.cs*src/CSharpTestLibrary/Properties/AssemblyInfo.cs
SRCSRV: end ------------------------------------------------
